ophidian/placement/Library.h

```diff
@@ -0,0 +1,66 @@
+#ifndef LIBRARY_H
+#define LIBRARY_H
+
+#include <ophidian/entity_system/EntitySystem.h>
+#include <ophidian/entity_system/Property.h>
+#include <ophidian/standard_cell/StandardCells.h>
+#include <ophidian/geometry/Models.h>
+#include <ophidian/util/Units.h>
+
+namespace ophidian {
+namespace placement {
+class Library
```

Comment:
This is a LEF_Library, correct? Maybe we should rename to Lef_Library to make it more clear. Therefore, it would be easier to map from the Lef parser. We even can add other important properties that are common in LEF libraries to make it more complete.